src/main/java/com/extremelyd1/gameboard/GameBoard.java

```diff
@@ -61,7 +61,19 @@ public GameBoard(Game game) {
     public void update() {
         // Clear all existing scores
         for (String entry : this.scoreboard.getEntries()) {
-            this.scoreboard.resetScores(entry);
+            Boolean entryStillExists = false;
```

Comment:
Again, please use the primitive `boolean` instead of the object wrapper.

Reply:
done